templates/webapi/admin/user/index.html.ep

```diff
@@ -19,6 +19,10 @@
                     <th>Name</th>
                     <th>Nick</th>
                     <th>Role</th>
+                    % if (is_admin) {
```

Comment:
Not very important but I personally prefer to stash a variable like `show_user_actions` instead of calling the helper all the time.